src/rules/order.js

```diff
@@ -277,7 +286,10 @@ function mutateRanksToAlphabetize(imported, alphabetizeOptions) {
   const groupRanks = Object.keys(groupedByRanks)
 
   const sorterFn = getSorter(alphabetizeOptions.order === 'asc')
-  const comparator = alphabetizeOptions.caseInsensitive ? (a, b) => sorterFn(String(a).toLowerCase(), String(b).toLowerCase()) : (a, b) => sorterFn(a, b)
+  const comparator =
+    alphabetizeOptions.caseInsensitive === 'invert' ? (a, b) => sorterFn(swapCase(String(a)), swapCase(String(b)))
+    : alphabetizeOptions.caseInsensitive ? (a, b) => sorterFn(String(a).toLowerCase(), String(b).toLowerCase())
```

Comment:
seems like this should be handled inside `getSorter` rather than here?

Reply:
Resolved with my new refactor commit.